static/js/reducers/videos.js

```diff
@@ -8,7 +8,7 @@ import type { Video } from "../flow/videoTypes";
 
 export const videosEndpoint = {
   name: "videos",
-  verbs: [GET, PATCH],
+  verbs: [GET, PATCH, "DELETE"],
```

Comment:
I think the latest version of redux-hammock includes `DELETE` now, but fine with me to make a separate issue/PR for updating it here later.

Reply:
Just created this issue: https://github.com/mitodl/odl-video-service/issues/337. I tried integrating the new version of redux-hammock but got an error that didn't make any sense.